plugins/services/src/js/components/ConfigurationMapDurationValue.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import ConfigurationMapValue from '../../../../../src/js/components/ConfigurationMapValue';
+import DateUtil from '../../../../../src/js/utils/DateUtil';
+import ValidatorUtil from '../../../../../src/js/utils/ValidatorUtil';
+
+const MULTIPLICANTS = {
+  'ms'  : 1,
+  'sec' : 1000,
+  'min' : 60000,
+  'h'   : 3600000,
+  'd'   : 86400000
+};
+
+/**
+ * Render a JSON object as a <ConfigurationMapValue>, within the
+ * appropriate formatting.
+ */
+class ConfigurationMapDurationValue extends React.Component {
+  render() {
+    let {defaultValue, multiplicants, units, value} = this.props;
+
+    // Bail early with default if empty
+    if (ValidatorUtil.isEmpty(value)) {
+      return <ConfigurationMapValue>{defaultValue}</ConfigurationMapValue>;
+    }
+
+    // Compose value multiplicants
+    let valueInMs = value * multiplicants[units];
+    let components = DateUtil.msToMultiplicants(valueInMs, multiplicants);
+
+    return (
+      <ConfigurationMapValue>
+        {`${value} ${units} (${components.join(', ')})`}
+      </ConfigurationMapValue>
+    );
+  }
+};
+
+ConfigurationMapDurationValue.defaultProps = {
+  defaultValue: <span>&mdash;</span>,
+  multiplicants: MULTIPLICANTS,
+  units: 'ms',
+  value: null
+};
+
+ConfigurationMapDurationValue.propTypes = {
+  defaultValue: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.node
+  ]),
+  multiplicants: React.PropTypes.object,
+  units: React.PropTypes.oneOf(Object.keys(MULTIPLICANTS)),
+  value: React.PropTypes.number.isRequired
```

Comment:
Since we have a default for this, let's remove the `isRequired` so it aligns with the other Configuration*Value components.